apps/nextjs-website/src/components/atoms/CtaCard/CtaCard.tsx

```diff
@@ -22,6 +25,7 @@ export type CtaCardProps = {
   readonly comingSoon?: boolean;
   readonly icon?: ReactNode;
   readonly children?: ReactNode | ReactNode[];
+  readonly products?: Pick<Product, 'name' | 'path'>[];
```

Comment:
Instead of using `Product` type i think is better to add an attribute like `tags` to CtaCard to manage those labels. 
Something like this:
```suggestion
  readonly tags?: { label: string; path?: string; }[];
```
This make component more agnostic and decouples evantual changes on Product type
PS. remember to manually update PR if you accept this suggestion because i have not update all occurences of this attribute